src/styles/blocks/card.scss

```diff
@@ -0,0 +1,103 @@
+* {
+  margin: 0;
+}
+
+.card {
+  display: flex;
+  color: #060b35;
+  background: #fff;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+  width: 198.4px;
```

Comment:
Avoid using pixel values for width unless it's strictly required by design. It's better to use percentages or responsive units to make the layout more flexible and responsive.